DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -2155,4 +2237,220 @@ internal class ContextMenuCommand
         /// </summary>
         public MethodInfo Method { get; set; }
     }
+
+    #region Default Help
+    /// <summary>
+    /// Represents the default help module.
+    /// </summary>
+    public class DefaultHelpModule : ApplicationCommandsModule
+    {
+        public class DefaultHelpAutoCompleteProvider : IAutocompleteProvider
+        {
+            public async Task<IEnumerable<DiscordApplicationCommandAutocompleteChoice>> Provider(AutocompleteContext context)
+            {
+                var options = new List<DiscordApplicationCommandAutocompleteChoice>();
+
+                var globalCommands = await context.Client.GetGlobalApplicationCommandsAsync();
+                var guildCommands = await context.Client.GetGuildApplicationCommandsAsync(context.Guild.Id);
+                var slashCommands = globalCommands.Concat(guildCommands)
+                .Where(ac => !ac.Name.Equals("help", StringComparison.OrdinalIgnoreCase))
+                .GroupBy(ac => ac.Name).Select(x => x.First());
+                var list = slashCommands.ToList();
+                var count = 0;
+                foreach (var sc in list)
+                {
+                    if (count < 25)
+                    {
+                        options.Add(new DiscordApplicationCommandAutocompleteChoice(sc.Name, sc.Name.Trim()));
+                        count++;
+                    }
+                    else
+                        continue;
```

Comment:
Technically not needed if the above is implemented but worth noting that a break would have been better. As the scenario noted above... if you had 50 items you'd be skipping over 26 items instead of immediately jumping out